tests/test_cli.py

```diff
@@ -1,10 +1,19 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-
+import os
 import pytest
 from click.testing import CliRunner
 from iotedgehubdev import cli
+from iotedgehubdev import configs
+
+VALID_DEVICECONNECTIONSTRING = os.environ['DEVICE_CONNECTION_STRING']
+
+
+def teardown_module():
```

Comment:
This method is never called. You can add a print inside to verify.

Reply:
retried. It is invoked.